src/internal.c

```diff
@@ -26940,7 +26940,7 @@ static int ParseCipherList(Suites* suites,
             }
         }
     }
-    while (next++); /* ++ needed to skip ':' */
+    while (next && (next++, 1)); /* ++ needed to skip ':' */
```

Comment:
Hi @alexsn we don't use comma expressions due to portability concerns. Please re-write. Also please share the shared to reproduce the sanitizer warning so we can setup testing for this.

Reply:
Ok, re-wrote to use a helper to do the increment. As far as reproducing the issue, I'll try to provide a minimal example tomorrow.